tests/testLeastSquares.py

```diff
@@ -42,36 +42,34 @@
 import lsst.pex.logging
 from lsst.afw.math import LeastSquares
 
-numpy.random.seed(500)
-
 lsst.pex.logging.getDefaultLog().setThresholdFor("afw.math.LeastSquares", -10)
 
-#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
-
+class LeastSquaresTestCase(lsst.utils.tests.TestCase):
 
-class LeastSquaresTestCase(unittest.TestCase):
+    def _assertClose(self, a, b, rtol=1E-5, atol=1E-8):
+        self.assertFloatsAlmostEqual(a, b, rtol=rtol, atol=atol, msg="\n%s\n!=\n%s" % (a, b))
 
-    def assertClose(self, a, b, rtol=1E-5, atol=1E-8):
-        self.assert_(numpy.allclose(a, b, rtol=rtol, atol=atol), "\n%s\n!=\n%s" % (a, b))
+    def _assertNotClose(self, a, b, rtol=1E-5, atol=1E-8):
+        self.assertFloatsNotEqual(a, b, rtol=rtol, atol=atol, msg="\n%s\n==\n%s" % (a, b))
 
-    def assertNotClose(self, a, b, rtol=1E-5, atol=1E-8):
-        self.assert_(not numpy.allclose(a, b, rtol=rtol, atol=atol), "\n%s\n==\n%s" % (a, b))
+    def setUp(self):
+        numpy.random.seed(500)
```

Comment:
I think it's too late for this ticket, but for future test improvements please also replace `import numpy` with `import numpy as np`